wherehows-web/app/services/banners.ts

```diff
@@ -0,0 +1,87 @@
+import Service from '@ember/service';
+import { computed, get } from '@ember/object';
+import { NotificationEvent } from 'wherehows-web/services/notifications';
+import { delay } from 'wherehows-web/utils/promise-delay';
+
+/**
+ * Expected properties to be found on a basic banner object added to our list
+ */
+export interface IBanner {
+  content: string;
+  type: NotificationEvent;
+  isExiting: boolean;
+  isDismissable: boolean;
+  iconName: string;
+}
+
+/**
+ * When creating a new banner, helps to determine whether that banner isDismissable
+ */
+const isDismissableMap: { [key: string]: boolean } = {
+  [NotificationEvent['info']]: true,
+  [NotificationEvent['confirm']]: true,
+  [NotificationEvent['error']]: false
+};
+
+/**
+ * When creating a new banner, helps to determine the kind of font awesome icon indicator will be on
+ * the left side of the banner
+ */
+const iconNameMap: { [key: string]: string } = {
+  [NotificationEvent['info']]: 'info-circle',
+  [NotificationEvent['confirm']]: 'exclamation-circle',
+  [NotificationEvent['error']]: 'times-circle'
+};
+
+export default class BannerService extends Service {
```

Comment:
As per above comment, can we add-in this functionality to the notifications service, probably maintain a separate queue for banners, since they can be rendered without respect to modals and dialogs. and also have `banner` NotificationType